README.md

```diff
@@ -659,14 +659,15 @@ end
 ## Testing with CarrierWave
 
 It's a good idea to test your uploaders in isolation. In order to speed up your
-tests, it's recommended to switch off processing in your tests, and to use the
+tests, it's recommended to switch off processing in your tests, disable SSRF protection, and to use the
```

Comment:
> Actually I don't recommend to do this. The users shouldn't do this without knowing what they're doing, unless they will be at risk.

Thank you for your comment. Do you mean there is risk of SSRF attack also during testing?

```suggestion
tests, it's recommended to switch off processing in your tests, and to use the file storage.
Also, you can disable SSRF protection at your own risk using the `skip_ssrf_protection` configuration.

In Rails you could do that by adding an initializer with:

```

What do you think about this explanation?

Reply:
My intention was to use this config during testing and only in testing to avoid outbound requests, because integration tests with webmock fails like explained in https://github.com/arkadiyt/ssrf_filter/issues/33

ssrf_filter performs DNS resolution, which is undetected by UnhandledHTTPRequestError or NetConnectNotAllowedError. Mocking as explained in https://github.com/arkadiyt/ssrf_filter/issues/33#issuecomment-783758611 might make it appear as if the test isn't making outbound requests, but that's not the case in reality.